docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -297,7 +298,8 @@ target directory you can specify the executable with the `-Dnative.image.path=`
 
 == Creating a container
 
-IMPORTANT: Before going further, be sure to have a working container runtime (Docker, podman) environment.
+IMPORTANT: Before going further, be sure to have a working container runtime (Docker, podman) environment. If you use Docker 
+on Windows you should share your project's drive at Docker Desktop file share settings.
```

Comment:
This is an interesting find. Can you clarify why it's needed?

Reply:
You will get "File not found" error until you share the drive. https://github.com/quarkusio/quarkus-quickstarts/issues/407

Reply:
That's interesting. @rsvoboda are you perhaps able to confirm this?

Reply:
https://docs.docker.com/docker-for-windows/#file-sharing

Reply:
I can confirm file sharing needs to be adjusted in Docker settings.

With default Docker Desktop for Windows installation I got this error:
```
Error: Invalid Path entry getting-started-1.0-SNAPSHOT-runner.jar
Caused by: java.nio.file.NoSuchFileException: /project/getting-started-1.0-SNAPSHOT-runner.jar
```

Reply:
Excellent, thanks @rsvoboda!